.github/workflows/install.yml

```diff
@@ -0,0 +1,74 @@
+name: Installation
+on:
+  push:
+    paths:
+      - "install/**"
+  pull_request:
+    paths:
+      - "install/**"
+
+jobs:
+  check:
+    name: check
+    runs-on: ubuntu-20.04
+    container: koalaman/shellcheck-alpine:stable
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: shell
+        run: shellcheck install/install.sh
+
+  shell:
+    name: "Install in ${{ matrix.shell }}"
+    runs-on: ubuntu-20.04
+    needs:
+      - check
+    strategy:
+      matrix:
+        shell:
+          - dash
+          - bash
+          - zsh
```

Comment:
IIRC zsh doesn't work, and it isn't POSIX

Reply:
zsh has an `emulate sh` mode that brings it closely enough to POSIX.

See https://github.com/MirBSD/mksh/blob/463cce09ddaa8b11f839d2e5747f85ee412b1a76/Build.sh#L34-L41 how to portably enable it (this is `mksh`’s build script, which basically has to run with any `sh`, POSIX or not/before)…